shared/graphql/schema.graphql

```diff
@@ -500,7 +500,23 @@ type AdminQueries {
   user: UserAdminQueries!
 }
 
+type MyRole {
+  name: String!
+}
+
+type Me {
+  name: String!
+  email: String!
+  isAdmin: Boolean!
+  role: MyRole!
+  roles: [MyRole!]!
+}
+
+union SwitchRoleResult = Me | OperationError
+
 type Query {
+  me: Me!
```

Comment:
are we sure this can't be called by anonymous?

Reply:
well, it probably can be, so we should make it nullable